Contest_API.md

```diff
@@ -1058,6 +1056,99 @@ clients with `admin` or `analyst` role. The `reaction` attribute
 is available to clients with `public` role only when the contest is
 not frozen.
 
+#### POST submissions
+
+To add a submission one can use the `POST` method on the submissions endpoint.
+The `POST` must include a valid JSON object with the same attributes the submission
+endpoint returns with a `GET` request with the following exceptions:
+
+* The attributes `id`, `team_id`, `time`, and `contest_time` are
+  optional. However, depending on the use case (see below) the server
+  may require attributes to either be absent or present, and should
+  respond with a 400 error code in such cases.
+* Since `files` only supports `application/zip`, providing the `mime` field is
+  optional.
```

Comment:
Is the server required to reject a POST that includes a mime that is not `application/zip`?

Reply:
As written the server isn't required to even look at the mime type, but would fail and reject later if the actual content wasn't a zip. IMHO this probably should be one of the failure requirements, but I would go ahead and merge and then discuss/PR this.

Reply:
See #18 for this.